planner/core/logical_plan_test.go

```diff
@@ -2196,8 +2196,8 @@ func (s *testPlanSuite) TestWindowFunction(c *C) {
 			result: "[planner:3591]Window 'w1' is defined twice.",
 		},
 		{
-			sql:    "select sum(a) over(w1), avg(a) over(w2) from t window w1 as (partition by a), w2 as (w1)",
```

Comment:
Why removed `sum(a) over (w1)` from this test?

Reply:
Because now we build the window function by iterating the map, so the test result may not stable if we do not remove it.